src/zenml/cli/__init__.py

```diff
@@ -556,6 +556,14 @@
 you originally registered it. (If you want to use your secrets manager, you
 should pass its name in with the `-x` option flag.)
 
+Keep in mind that once you register a stack, its artifact store and
+metadata store will be coupled together, and using these instances in other
+pairings will not be allowed. This is done to avoid mismatches between
+these two stores. If you want to bypass this check when you are
```

Comment:
```suggestion
these two stores. If you want to override this check when you are
```
override sounds a bit better to me

Reply:
Changed it!